production/kubernetes/install-loki.sh

```diff
@@ -35,7 +35,7 @@ MANIFEST_URL=${MANIFEST_URL:-https://raw.githubusercontent.com/grafana/agent/${M
 LOKI_USERNAME_SET=0
 LOKI_PASSWORD_SET=0
 
-while getopts "l:u:p:" opt; do
+while getopts "h:u:p:" opt; do
```

Comment:
This was a bug in the Loki script. I know it should be its own PR, but I'm being lazy by sneaking in a fix.